composer/trainer/mosaic_fsdp_utils.py

```diff
@@ -86,6 +86,7 @@ def get_mixed_precision(precision, mixed_precision='DEFAULT', keep_low_precision
             pass
         elif mixed_precision == 'DEFAULT':
             param_dtype = _get_torch_dtype(precision)
+            reduce_dtype = torch.float32
```

Comment:
```suggestion
            reduce_dtype = torch.get_default_dtype()
```

This might be more correct:
doc: https://pytorch.org/docs/stable/generated/torch.get_default_dtype.html

Reply:
granted this warrants changing docs...